pages/detailpage/[id].js

```diff
@@ -0,0 +1,49 @@
+import { useRouter } from "next/router";
+/*import { campingTrips } from "../../data/campingTrips";*/
+import { Container, Card, Header, Footer } from "../../styles/styles";
+import Image from "next/image";
+import Link from "next/link";
+
+export default function DetailPage({ campingTrips }) {
+  const router = useRouter();
+  const { id } = router.query;
+
+  let trip = campingTrips.find((campingTrip) => campingTrip.id === id);
+  console.log(campingTrips);
+
+  return (
+    <>
+      <Header>
+        <h1>Check the details fo your trips</h1>
+      </Header>
+      <Container>
+        <Card>
+          <h2>{trip && trip.day}</h2>
```

Comment:
You could try if you can delete the "trip &&" in all the lines. I'm not sure, but it could work and would make the code look a bit nicer :)